ldap/servers/plugins/posix-winsync/posix-group-func.c

```diff
@@ -88,11 +88,11 @@ getEntry(const char *udn, char **attrs)
         if (result != NULL) {
             return result; /* Must be freed */
         } else {
-            slapi_log_err(SLAPI_LOG_PLUGIN, POSIX_WINSYNC_PLUGIN_NAME,
-                          "getEntry: %s not found\n", udn);
+            slapi_log_err(SLAPI_LOG_ERR, POSIX_WINSYNC_PLUGIN_NAME,
```

Comment:
I would prefer that the caller of getEntry decides if it is an error or not (that the entry does not exist) and then logs  the appropriate message.
Like you did in getMembershipFromDownward and modGroupMembership

Reply:
Sure, I'll tweak that.